### PR TITLE
feature-team1ticket007

### PR DESCRIPTION
**Pull request by JamesJose15**
New feature add ticket007

### CODE REVIEW

File: src/components/UploadComponent/ItemUpload/itemUpload.styles.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { makeStyles } from "@material-ui/core";
 
 const useStyles = makeStyles(theme => ({
     container: {
-        padding: '10px'
-    }
+        padding: '10px',
```
Comment (Ditmar) [inline src/components/UploadComponent/ItemUpload/itemUpload.styles.js:5]: use rem instead pixel units

File: src/components/UploadComponent/index.jsx [hunk 2]
```diff
@@ -1,43 +1,13 @@
-import { useContext, useState } from "react";
-import { post } from "../../async/api";
-import { LoaderSpinnerContext } from "../../context/LoaderSpinnerContext";
-import useUpload from "../../hooks/useUpload";
-import ItemUpload from "./ItemUpload";
+import FileUpload from "./FileUpload";
+//import ItemUpload from "./ItemUpload"
```
Comment (Ditmar) [inline src/components/UploadComponent/index.jsx]: remote the comment

File: src/components/UploadComponent/FileUpload/index.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+
+import useStyles from "./fileUpload.styles";
+import useUpload from "../../../hooks/useUpload";
+import { DeleteForever, PictureAsPdf } from "@mui/icons-material";
+
+const FileUpload = ({onCancel, name, type, size, onUpload}) => {
+    const classes = useStyles();
+    const upload =  useUpload(1)
+    function handleClick(){
+        alert(upload.file.name)
```
Comment (Ditmar) [inline src/components/UploadComponent/FileUpload/index.jsx:10]: Why do you use an alert here that is not part of the scope of the ticket

File: src/components/UploadComponent/FileUpload/index.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    function handleClickDelete(){
+        upload.onCancelUpload()
+    }
+
+    function CardInfo(){
+        return(
+            <div className={classes.cardInfo}>                    
+                <PictureAsPdf style={{ color: 'orange' , width: "50px", height:"50px" }} /> <br/> {upload.file.name}  <br/>  {upload.file.type} <br/> {upload.file.size} MB
```
Comment (Ditmar) [inline src/components/UploadComponent/FileUpload/index.jsx:20]: The color needs to come from the palette of colors from here https://github.com/Ditmar/plagiarist-g1/blob/master/src/theme/theme.js 
you can access to the color using the styles.

File: src/components/UploadComponent/FileUpload/fileUpload.styles.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        height: '100%',
+        outline:"none",
+        opacity:"0"
+    },
+    button:{
+        margin: "10px",
+        border:" solid 1px gray",
+        borderRadius : "5px",
+        color :"aliceblue",
+        background:"#239f98",
```
Comment (Ditmar) [inline src/components/UploadComponent/FileUpload/fileUpload.styles.js:29]: Please don't hardcode the colors, use the theme.
`theme.palette.fourth.main`
This is only an example, you can edit the file https://github.com/Ditmar/plagiarist-g1/blob/master/src/theme/theme.js 
and add the colors that you need.

**Comment by Ditmar** [comment]
Nice work @JamesJose15 , only a few comments please review that and your pr will be merged to the master branch

**Comment by Ditmar** [comment]
Something more is neceary to rebase the pr 
`
please rebase your dev-branches
if you don't remember how do merge. 
you can use 
in the master branch 
git pull origin master 
change to your feat branch
git rebase -X ours master
after that is necessary to push the changes in the remote branch
for that you can use
git push -f
`

**Comment by JamesJose15** [comment]
@Ditmar already fixed the conflicts I had, could you review my ticket please

**Review by Ditmar — CHANGES_REQUESTED** [review]

**Review by Ditmar — APPROVED** [review]

**Pull request merged**